atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/SchemaHotspottingTest.java

```diff
@@ -34,46 +34,122 @@
     @Rule
     public TemporaryFolder temporaryFolder = new TemporaryFolder();
 
+    private static final String HOTSPOTTING_TABLE = "HotspottingTable";
+    private static final String IGNORED_HOTSPOTTING_TABLE = "IgnoredHotspottingTable";
+    private static final String HOTSPOTTER_ROW_COMPONENT = "Hotspotter";
```

Comment:
These would probably be better as `TABLE_NAME`, `ROW_COMPONENT_NAME` since the values don't really make any difference.